katex.webpack.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * This is the webpack entry point for KaTeX. As flow[1] and jest[2] doesn't support
+ * CSS modules natively, a separate entry point is used and it is not flowtyped.
+ *
+ * [1] https://gist.github.com/lambdahands/d19e0da96285b749f0ef
+ * [2] https://facebook.github.io/jest/docs/en/webpack.html
+ */
+import './static/katex.less';
+import katex from './katex.js';
+
+module.exports = katex;
```

Comment:
This is so that the build file will be commonjs?

Reply:
@kevinbarabash Yes, I think we can (and should) switch to ES6 module `export`, but it seems it's out of scope of this PR.

Reply:
@kevinbarabash Actually, build file format is specified in webpack configuration `output.libraryTarget` and it's set to UMD (Universal Module Definition). Changed to ES6 module.

Reply:
I believe this would be a breaking change. Users would just need to be aware of that.